content/marketing/content/editorial/editorial-strategy.md

```diff
@@ -72,24 +72,15 @@ According to a question asked in Slack, our devs would trust the information com
 
 Below is a non-exhaustive, in-progress list of learnings we've gathered from reading the above blogs.
 
-## Balance depth and compactness
+#### Subject matter, style, and tone
```

Comment:
@nickmyyz There was so much great stuff to digest here so I took a crack at grouping some of these into sections to make it easier to digest!